orbit/pkg/platform/platform.go

```diff
@@ -0,0 +1,173 @@
+package platform
+
+import (
+	"errors"
+	"fmt"
+	"os"
+	"path/filepath"
+	"strconv"
+	"strings"
+
+	"github.com/mitchellh/go-ps"
+	"github.com/rs/zerolog/log"
+	gopsutil_process "github.com/shirou/gopsutil/v3/process"
+)
+
+var ErrProcessNotFound = errors.New("process not found")
+
+// Kills a single process by its name
+func KillProcessByName(name string) error {
+	if name == "" {
+		return fmt.Errorf("process name should not be empty")
```

Comment:
nit: here and in some places below: `errors.New` is preferred when you're not wrapping an error.

```suggestion
		return errors.New("process name should not be empty")
```

Reply:
I didn't know that  `lint-go` was a thing. I'm running it now before pushing new code :). Thanks for the advice!